frontend/src/bundles/workouts/components/workout-stats/workout-stats.tsx

```diff
@@ -23,7 +23,8 @@ const WorkoutStats: React.FC<Properties> = ({ workout }): JSX.Element => {
         workoutEndedAt,
     } = workout;
 
-    const workoutDistance = steps ?? metersToKilometers(distance);
+    const workoutDistance =
+        steps && steps !== 0 ? steps : metersToKilometers(distance);
```

Comment:
const workoutDistance = steps || metersToKilometers(distance);

just curious, is this disallowed by linter? no need to adjust